presto-main/src/main/java/com/facebook/presto/tracing/TracingConfig.java

```diff
@@ -24,6 +24,13 @@
     {
         public static final String NOOP = "noop";
         public static final String SIMPLE = "simple";
+        public static final String OTEL = "otel";
+    }
+
+    public static class ContextPropagator
```

Comment:
Can we delete this class from this file?

Reply:
Done